src/d3a/d3a_core/user_profile_handler.py

```diff
@@ -16,13 +16,12 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 import os
-from datetime import datetime
-from typing import Dict
 import uuid
+from datetime import datetime, timezone
+from typing import Dict
 
-import d3a.constants
+import pendulum
```

Comment:
minor, and please leave it as it is a crucial PR:
Please homogenise the imports (there are already occurrences with `from pendulum import ...`

Reply:
no problem I added it now. It is more or less one more CI run after all